### PR TITLE
alan/coo-1004-update-docs-auto-cookie-scanning

### PR DESCRIPTION
**Pull request by alanbiju79**

### CODE REVIEW

File: images/scan/.DS_Store
Comment (danielpaul) [inline images/scan/.DS_Store:1]: Delete da store hidden files

File: docs/features/vendor-scanning.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Vendor Scanning"
+description: "Understand how CookieChimp scans your website for vendors."
+---
+
+<img src="/images/scan/scan.png" alt="Scan page" />
+
+We scan your website to find the vendors and storage items used on your site.
+We check what storage items were set before and after consent was granted.
```
Comment (danielpaul) [inline docs/features/vendor-scanning.mdx:9]:
```suggestion
We detect services and storage items set before and after consent was granted.
```

File: docs/features/vendor-scanning.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+If a service runs, that are not supposed to run before consent is granted, we will flag this as `Ran before consent`.
```
Comment (danielpaul) [inline docs/features/vendor-scanning.mdx:11]:
```suggestion
If a service that's not part of a required category runs before consent is granted, we will flag this as `Ran before consent`.
```

File: docs/features/vendor-scanning.mdx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+If we detect storage items from a service that is not shown in your banner, we will flag the service as `Not shown in banner`.
+
+If we detect storage items that are not in your banner and we cannot match to a vendor/service, we will flag these as `Not part of a category or service`.
+
+If everything is set up correctly then we flag the service as `All good`.
```
Comment (danielpaul) [inline docs/features/vendor-scanning.mdx:17]: We can make all of these be bullet points and the status first and the description after it

File: docs/features/vendor-scanning.mdx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+We also check if users can update their consent with our privacy trigger button.
```
Comment (danielpaul) [inline docs/features/vendor-scanning.mdx:19]:
```suggestion
We also check if users can update their preferences after their initial consent. 
```

CMS be via our privacy trigger or a custom link

File: docs/features/vendor-scanning.mdx [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+Accounts on free plans are limited to 1 scan per day.
+Accounts on paid plans have access to 5 scans per day.
+
+## Auto Scanning
+
+Auto scanning can be enabled through the Vendor Scanner integration.
+
+<img src="/images/scan/vendor-scanner.png" alt="Scan page" />
+
+Auto scanning can be enabled or disabled. The frequency of the scan can set to 'Weekly' or 'Monthly'.
```
Comment (danielpaul) [inline docs/features/vendor-scanning.mdx:36]:
```suggestion
```

File: docs/features/vendor-scanning.mdx [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+Automatic weekly scanning is only available to accounts on paid plans.
+Accounts on free plans only have access to automatic monthly scanning.
```
Comment (danielpaul) [inline docs/features/vendor-scanning.mdx:38]: On free plans we only allow mannual scanning no?

**Review by danielpaul — CHANGES_REQUESTED** [review]

File: docs/features/vendor-scanning.mdx (hunk 4 above)
Comment (alanbiju79) [reply docs/features/vendor-scanning.mdx]: what does 'CMS be via our privacy trigger or a custom link' mean?

**Review by danielpaul — APPROVED** [review]

**Pull request merged**